normandy/control/tests/actions/controlActions.js

```diff
@@ -70,79 +74,84 @@ describe('controlApp Actions', () => {
     const expectedAction = { type: actionTypes.RECIPES_RECEIVED, recipes: fixtureRecipes }
     spyOn(window, 'fetch').and.returnValue(successPromise(fixtureRecipes));
 
-    return store.dispatch(controlActions.makeApiRequest('fetchAllRecipes')).then(() => {
+    return store.dispatch(actionTypes.makeApiRequest('fetchAllRecipes'))
+    .then((response) => {
+      store.dispatch(actionTypes.recipesReceived(response));
```

Comment:
The test description says that we're testing that this action is dispatched on a successful fetch, but you've changed the API so that this action isn't automatically dispatched. Manually dispatching it in the test and then asserting below that is was dispatched doesn't test anything.

At this point this test is about testing that the `makeApiRequest` action calls `window.fetch` with the proper arguments. The description should be updated and the other assertions removed.